charts/docker-registry-ui/values.yaml

```diff
@@ -170,6 +170,16 @@ registry:
   additionalSpec: {}
   # Extra Environmental Variables for Registry
   extraEnv: []
+  # Add Basic Auth to Registry.
+  basicAuth:
+    # Enable basic auth for Registry.
+    enabled: false
+    # Basic auth realm.
+    realm: Docker registry
+    # Full path for htpasswd file. Note that filename should match the secret key.
+    htpasswdPath: /auth/htpasswd
```

Comment:
Move to `/etc/docker/registry/htpasswd` or `/etc/docker/registry/auth/htpasswd` ?

Reply:
sounds good to me, for making it more clean in one place 👍